Shared/Samples/Show viewshed from point in scene/ShowViewshedFromPointInSceneView.swift

```diff
@@ -37,8 +37,7 @@ struct ShowViewshedFromPointInSceneView: View {
                     .background(.thinMaterial, ignoresSafeAreaEdges: .horizontal)
             }
             .toolbar {
-                ToolbarItemGroup(placement: .bottomBar) {
-                    Spacer()
+                ToolbarItem(placement: .bottomBar) {
```

Comment:
Is there a reason this is centered (in the image below) whereas "Light and Shadow" and "Orbit Around Plan" buttons are right-aligned?

Reply:
My understanding is that is intended SwiftUI behavior. Starting on iOS 16, if a picker is centered, the following happens. That is why we decided that to leave the item alignment up to the system as there could be more breaking changes like this in the future.

https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/09201655-8b03-44f3-919c-393a20ff7356